### DISCUSSION TITLE
Project HÆLTH Blog #5

### DISCUSSION THREAD

**Discussion by ldcdorn** [ldcdorn/haelth#52] [General]
# Project HÆLTH 🌱
This week Project HÆLTH has reached a major milestone, the first running demo of our app 🥳

Have a look at this screenshot of our initial concept:

![haelth_ui](https://github.com/user-attachments/assets/3e0cca97-495f-4b1e-afc3-edd30c271189)

Major thanks to all team members for contributing to this release.

Here's what everyone contributed individually:
@SidMllr, our PO, contributed massively to the initial wire frame and UI/UX of Project HÆLTH, we would have been able to publish this release without his dedication, technical ability, and superb leadership. @j1tbk took over the role of lead programmer for this release, his ability to handle all technical details and tackle challenging programming problems is invaluable to our team. @OmarMoujahed kept the infrastructure running while the rest was focusing on development. We faced some issues with collaboration on Android Studio, but Omar with his broad knowledge of CI/CD was able to sort that out in no time. I (@ldcdorn) for my part as Scrum Master of the team made sure that all the tangential issues that arose didn't affect the Development team. I made sure that everyone had a comfortable environment to work in and that the development progressed smoothly.

**Comment by themsgcentre** [comment]
Die Demo sieht schon mal sehr gut und übersichtlich aus! Besonders die bereits vorgenommene Unterteilung in die drei Bereiche Mindfulness, Nutrition und Fitness ist bemerkenswert und bietet eine klare Struktur für das Projekt.

Ein kleiner Hinweis: Es gibt noch einen Rechtschreibfehler, es wurde „mindfulness“ als „mindfullness“ geschrieben.

Insgesamt ein toller Start – wir sind gespannt auf die nächsten Schritte!

Viele Grüße,
Projekt TDC (HM)

**Comment by Anastylos** [comment]
Das Design sieht sehr übersichtlich aus, der Rechschreibfehler wurde ja bereits angesprochen. Schön wäre es gewesen ein Mockup Icon o.ä. zentral zu plazieren, sodass man sich schon besser vorstellen kann wie die fertige Seite etwa aussehen wird. Ich bin gespannt auf den Fortschritt eures Projektes.

liebe Grüße von Team Jump and run

**Comment by oskarkreutzer** [comment]
Euer Fortschritt sieht vielversprechend aus, und das klare Design mit den drei Hauptbereichen ist eine gute Basis! Für die visuelle Präsentation könnte es hilfreich sein, ein zentrales Mockup-Icon einzubinden, das die Nutzeroberfläche deutlicher skizziert. Auch die Rechtschreibkorrektur („mindfullness“ zu „mindfulness“) ist wichtig, um die Professionalität zu bewahren. Insgesamt eine starke erste Demo – wir freuen uns darauf zu sehen, wie sich das Projekt weiterentwickelt!

Mit lieben Grüßen verbleibt
das GymTrackerX Team
